demisto_sdk/common/hook_validations/release_notes.py

```diff
@@ -0,0 +1,152 @@
+        lines of context.
+
+        Returns:
+            str. empty string if no changes made or no origin/master branch, otherwise full difference context.
+        """
+        return run_command(F'git diff --unified=100 '
+                           F'origin/master {self.release_notes_path}')  # type: str
+
+    def is_release_notes_changed(self):
+        """Validates that a new comment was added to release notes.
+
+        Returns:
+            bool. True if comment was added, False otherwise.
+        """
+        # there exists a difference between origin/master and current branch
+        if self.master_diff:
+            diff_releases = self.master_diff.split('##')
+            unreleased_section = diff_releases[1]
+            unreleased_section_lines = unreleased_section.split('\n')
+
+            adds_in_diff = 0
+            removes_in_diff = 0
+
+            for line in unreleased_section_lines:
+                if line.startswith('+'):
+                    adds_in_diff += 1
+                elif line.startswith('-') and not re.match(r'- *$', line):
+                    removes_in_diff += 1
+
+            # means that at least one new line was added
+            if adds_in_diff - removes_in_diff > 0:
+                return True
+
+        print_error(F'File {self.release_notes_path} is not formatted according to '
+                    F'release notes standards.\nFix according to {self.LINK_TO_RELEASE_NOTES_STANDARD}')
+        return False
+
+    def is_valid_release_notes_structure(self):
+        """Validates that the release notes written in the correct manner.
+
+        one_line_release_notes_regex meaning:
+            Option #1:
+                starts with a letter or number
+                ends with '.'
+            Option #2:
+                starts with '-' with none or one trailing spaces.
+
+        multi_line_release_notes_regex meaning:
+            Option #1:
+                starts with tab or two to four spaces, then '-' ,then space
+                each line ends with '.'
+            Option #2:
+                starts with '-' with none or one trailing spaces.
+
+        Returns:
+            bool. True if release notes structure valid, False otherwise
+        """
+        one_line_release_notes_regex = r'[a-zA-Z0-9].*\.$|- ?$'
+        multi_line_release_notes_regex = r'(\t+| {2,4})- .*\.$|- ?$'
+
+        release_notes_comments = self.latest_release_notes.split('\n')
+
+        if not release_notes_comments[-1]:
+            release_notes_comments = release_notes_comments[:-1]
+
+        if len(release_notes_comments) == 1:
+            if not re.match(one_line_release_notes_regex, self.latest_release_notes):
+                print_error(F'File {self.release_notes_path} is not formatted according to '
+                            F'release notes standards.\nFix according to {self.LINK_TO_RELEASE_NOTES_STANDARD}')
+                return False
```

Comment:
Same as above, let's be specific

Reply:
Its difficult to do, since I cant know what the user has wanted to do.
I know what could have gone wrong, but not what the issue is.